build.sbt

```diff
@@ -88,7 +88,7 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
       _.filter(f => Set("JsonTest.scala", "PaigesTest.scala").contains(f.getName))
     },
     libraryDependencies ++= Seq(
-      "org.scalatest" %%% "scalatest" % "3.1.0-RC3" % Test
```

Comment:
it seems that `scalatest_native` was not published... not sure if this is intended. https://github.com/scalatest/scalatest/issues/1741

Reply:
I missed that scalatest was published against a newer version of scala native.